tests/e2e/workloads/bdi/fixtures.py

```diff
@@ -0,0 +1,254 @@
+import pytest
+import os
+import logging
+from ocs_ci.utility.utils import run_cmd, clone_repo, exec_cmd
+from ocs_ci.ocs import machine, node, ocp, constants
+from ocs_ci.framework import config
+import tempfile
```

Comment:
please move to be above ocs_ci import statements

Reply:
Done